core/templates/components/button-directives/hint-and-solution-buttons.component.spec.ts

```diff
@@ -133,7 +131,7 @@ describe('HintAndSolutionButtonsComponent', () => {
       }),
       RecordedVoiceovers.createEmpty(),
       writtenTranslationsObjectFactory.createEmpty(),
-      'content');
+      'content', null);
```

Comment:
Type will mismatch here, inject `audioTranslationLanguageService` and then use it here.Ditto elsewhere

Reply:
done